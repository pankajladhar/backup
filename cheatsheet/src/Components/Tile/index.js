import React from 'react';

const Tile = (props) => {
    return(
        <div className="Tile">Tile works</div>
    )
}

export default Tile;