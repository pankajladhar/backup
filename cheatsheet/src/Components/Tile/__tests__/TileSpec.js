import React from 'react';
import renderer from 'react-test-renderer';
import Tile from './../index';

describe('Tile', () => {
    it('renders correctly', () => {
        let tree = renderer.create(
            <Tile />
        );
        expect(tree.toJSON()).toMatchSnapshot();
    });
});