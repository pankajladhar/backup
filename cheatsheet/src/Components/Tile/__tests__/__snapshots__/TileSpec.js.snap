// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tile renders correctly 1`] = `
<div
  className="Tile"
>
  Tile works
</div>
`;
